tests/affiliatedcertification/tests/affiliated_certification_container.go

```diff
@@ -121,33 +121,148 @@ var _ = Describe("Affiliated-certification container certification,", func() {
 
 	// 46566
 	It("certifiedcontainerinfo field exists in tnf_config but has no value [skip]", func() {
-		Skip("Under development")
+		By("Add container information to tnf_config.yml")
```

Comment:
Please change to By("Add container information to " + globalparameters.DefaultTnfConfigFileName)